### ISSUE TITLE
neuroHarmonize two datasets

### ISSUE THREAD

**Issue by eliauk07yz** [rpomponio/neuroHarmonize#58]
Hello, if I have two datasets, train and test, I use neuroHarmonizes for each of these datasets, removing the site. Can the data from the two neuroHarmonize models be analyzed together after the removal of the site?

**Comment by rpomponio** [comment]
Without understanding the specific context, I think the answer is generally "yes".

Many users wish to remove site effects so that they may analyze the pooled dataset in some kind of machine learning application.

Note: if your test set images are on the same subjects, only later in time, you will probably want to look into [longitudinal ComBat](https://doi.org/10.1016/j.neuroimage.2020.117129).

Please feel free to re-open this issue at any time.

**Comment by eliauk07yz** [comment]
I mean there are two datasets train and test that don't have the same site, and I used the two NeuroHarmonize models to remove the site effect separately. Can I count them together after we do that? Or do I have to Harmonize these two datasets together？

**Comment by rpomponio** [comment]
The purpose of harmonization is to remove site effects so that data from different sites can be analyzed together. To accomplish this, all sites must be part of the same training dataset.

If a site is not in your training dataset, you will not be able to harmonize data from that site, full stop.

Therefore I recommend that you re-define your training set so as to include samples from both sites. In fact, you may wish to include all samples in the training set and skip the test set. The test set is simply for samples that you do not wish to influence the harmonization parameters (for example, samples from diseased subjects, or repeated scans of the same subject).
